### PR TITLE
Make features actions sticky while scrolling

### LINKED ISSUE #6724: Recommendation UX Improvement in Features sections / Admin
Once user going thru the Feature list, select required components he need to goes up till the beginning to enable / disable them.
Recommendation to add btn at the end.

image

### PR DESCRIPTION
**Pull request by hishamco**
Fixes #6724

According to the discussion with @Skrypt in #6857 we agreed that we shall use a sticky bar on scrolling for a very long pages instead of scroll to top button, here is how it look like:

Sticky

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Features/Views/Admin/Features.cshtml [hunk 1]
```diff
@@ -2,36 +2,31 @@
...
-                <a class="dropdown-item" href="#" data-action="@nameof(FeaturesBulkAction.Disable)">@T["Disable"]</a>
-                <a class="dropdown-item" href="#" data-action="@nameof(FeaturesBulkAction.Enable)">@T["Enable"]</a>
-                <a class="dropdown-item" href="#" data-action="@nameof(FeaturesBulkAction.Toggle)">@T["Toggle"]</a>
-            </div>
-        </div>
-    </div>
 </div>
-
-<form autocomplete="off">
+<form autocomplete="off" class="sticky">
```
Comment (Skrypt) [inline src/OrchardCore.Modules/OrchardCore.Features/Views/Admin/Features.cshtml:10]: Can you try with `position-sticky` instead?

File: src/OrchardCore.Themes/TheAdmin/Assets/scss/TheAdmin.scss [hunk 2]
```diff
@@ -1392,4 +1392,11 @@ input[type=password]::-ms-reveal,
 input[type=password]::-ms-clear
 {
     display: none;
+}
+
+.sticky {
+    position: -webkit-sticky;
```
Comment (Skrypt) [inline src/OrchardCore.Themes/TheAdmin/Assets/scss/TheAdmin.scss:1398]: maybe rename this to .action-bar and use position-sticky from BS4. Remove the position: -webkit-sticky and position sticky.

File: src/OrchardCore.Modules/OrchardCore.Features/Views/Admin/Features.cshtml (hunk 1 above)
Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Features/Views/Admin/Features.cshtml:10]: I didn't know that bootstrap tried to simulate all the things ;)

File: src/OrchardCore.Themes/TheAdmin/Assets/scss/TheAdmin.scss (hunk 2 above)
Comment (hishamco) [reply src/OrchardCore.Themes/TheAdmin/Assets/scss/TheAdmin.scss:1398]: Make sense

**Comment by hishamco** [comment]
I will address your feedback when I wake up, I'm sleepy ;)

**Comment by hishamco** [comment]
I'm done!!

**Review by Skrypt — APPROVED** [review]

**Comment by deanmarcussen** [comment]
Now that is much nicer @hishamco Good job

**Comment by hishamco** [comment]
I will do the same thing for Content Items page, but we could do it for other pages if we want

**Pull request merged**